wocg/create_project.py

```diff
@@ -77,13 +80,19 @@ def get_new_project(project_name, repository):
 
 def get_new_component(
         project, repository, branch, addon_name, tmpl_component_slug,
-        addons_subdirectory=None):
+        addons_subdirectory=None, git_export_base_url=None):
     po_file_mask = '{}/i18n/*.po'.format(addon_name)
     pot_filepath = '{addon_name}/i18n/{addon_name}.pot'.format(
         addon_name=addon_name)
+    git_export = ''
     if addons_subdirectory:
         po_file_mask = os.path.join(addons_subdirectory, po_file_mask)
         pot_filepath = os.path.join(addons_subdirectory, pot_filepath)
+    if git_export_base_url:
+        git_export = urljoin(
+            git_export_base_url,
+            os.path.join('git', project.name)
```

Comment:
do not mix urljoin and join (on windows join users \, and url are always with /)